backend/prisma/migrations/20211221182939_add_score_entry/migration.sql

```diff
@@ -0,0 +1,22 @@
+/*
+  Warnings:
```

Comment:
außerdem würde ich es cool finden, wenn du zwischen datum und Uhrzeit noch einen unterstrich machst - ist aber natürlich nur reine Geschmacksache :)
ich finde aber `20211222_083057` etwas leserlicher als `20211222083057` - dann sehe ich zumindest gleich das datum

Reply:
Dies wird vom Migrations-Tool von Prisma gehandelt.

Reply:
ah ok du lässt dir diese Dateien generieren? verstanden.
deshalb schreibe ich meine Migrationsfiles immer selbst, um sowas wie z.b. das Naming "in der hand" zu haben :)